component/common/src/main/java/org/exoplatform/management/common/AbstractOperationHandler.java

```diff
@@ -148,7 +148,8 @@ protected final Identity getIdentity(String id) {
         return spaceIdentity;
       }
     } catch (Exception e) {
-      log.error("Error while retrieving identity: ", e);
+      // issue : We got a log error with a stack trace of the exception.
+      log.warn("Failed to retrieve identity from id : "+id);

```

Comment:
BTW please use the {} notation to build the log message it more efficient that concatenate a string